packages/portfolio-contract/src/planner.exo.ts

```diff
@@ -109,6 +112,20 @@ export const preparePlanner = (
         portfolioPlanner.submitVersion(policyVersion, rebalanceCount);
         portfolioPlanner.resolveFlowPlan(flowId, planOrSteps);
       },
+      rejectPlan(
+        portfolioId: number,
+        flowId: number,
+        reason: string,
+        policyVersion?: number,
+        rebalanceCount?: number,
+      ) {
+        trace('TODO: reject plan', { portfolioId, flowId, reason });
+        const { planner: portfolioPlanner } = getPortfolio(portfolioId);
+        if (policyVersion !== undefined && rebalanceCount !== undefined) {
+          portfolioPlanner.submitVersion(policyVersion, rebalanceCount);
+        }
```

Comment:
The planner should always send `policyVersion` and `rebalanceCount`. Is their optionality intended to support another interaction, such as manual user cancellation? A comment here is probably warranted.

I'd also prefer `policyVersion` to never be ignored if provided:
```suggestion
        if (policyVersion !== undefined) {
          portfolioPlanner.submitVersion(policyVersion, rebalanceCount!);
        }
```

Reply:
> A comment here is probably warranted.

um. yeah. what was past me thinking?

Reply:
I suspect I made them optional because I didn't want to think about them while writing tests.

ah... perhaps it's LLM gorp that I didn't review sufficiently. Thanks for catching it.

I'm making them required.